verilog/CST/net_test.cc

```diff
@@ -149,15 +149,6 @@ TEST(GetIdentifiersFromNetDeclarationTest, EmptySource) {
   EXPECT_TRUE(net_declarations.empty());
 }
 
-TEST(GetIdentifiersFromNetDeclarationTest, NoNetIdentifier) {
```

Comment:
Now that `sin` is not a special token anymore, this will not work as expected.

(Also, I have honestly no idea what this test is testing for)